src/database/votePost.js

```diff
@@ -1,9 +1,8 @@
 import { db } from "database";
 import firebase from "firebase";
 
-export const voteForOption = ({ optionName, postId }) => {
-  const increment = firebase.firestore.FieldValue.increment(1);
+export const voteForOption = ({ optionName, postId }, uid) => {
```

Comment:
`uid` can go inside the same object as the rest of the parameters. The object parameter syntax is so that your function calls become more meaningful.

For example:
`setUser(true)` is not meaningful coz you don't know wtf true is for

As oppposed to:
`setUser({ isUserAdmin: true })` is more readable.

This is the advantage of using object parameters

Reply:
done